src/server/utils/settings-manager/settings-manager.ts

```diff
@@ -17,7 +17,13 @@
 
 import { Dataset, External } from "plywood";
 import { Logger } from "../../../common/logger/logger";
-import { AppSettings } from "../../../common/models/app-settings/app-settings";
+import {
```

Comment:
Look - how many responsibilities for app-settings.

Reply:
See, OOP is good at hiding complexity, not solving it :)

Reply:
I'm going to do second iteration in review :)